src/api/fetch/GetPromise.ts

```diff
@@ -0,0 +1,24 @@
+import HttpStatus from '../HttpStatus';
```

Comment:
Lag gjerne litt tester på denne. Synes den har lett for å knekke når vi begynner å klå på den i ettertid.

Reply:
La inn tester her.